docs/spec.md

```diff
@@ -1358,7 +1358,8 @@ implementation-defined as well.
 
 Expands the dimensions and/or rank of an input tensor by duplicating the data
 in the `operand` tensor and produces a `result` tensor. More formally,
-`result[result_index] = operand[operand_index]` where:
+`result[result_index] = rank(operand) = 0 ? operand : operand[operand_index]`
```

Comment:
lgtm. Was wondering if we should use the more pythonic syntax or a ternary operator, but elsewhere in the spec prefers ternary so this is consistent (i.e. `max_element = rank(max) = 0 ? max : max[result_index]`)